.github/workflows/cleanup.yaml

```diff
@@ -52,7 +52,7 @@ jobs:
         run: |
           snow sql \
             -q "
-              SET safe_environment_identifier = (SELECT REPLACE('$GITHUB_REF_NAME', '-', '_'));
+              SET safe_environment_identifier = (SELECT REPLACE('${{ github.event.ref }}', '-', '_'));
```

Comment:
I'm looking at examples of `github.event.ref` and it seems to indicate that it includes the full reference path not just the branch name e.g `refs/heads/main`?

Reply:
@rxu17 The naming convention would make more sense if it did that, but you can see its actual behavior in this line where we use `github.event.ref` to substitute the namespace/branch name: https://github.com/Sage-Bionetworks/recover/actions/runs/9963030161/job/27528123692#step:3:7

Reply:
Ah i didn't notice this subtle difference before. Does the behavior of `github.event.ref` change if it's used in the conditional context or it just spits out the exact same thing as `github.ref_name` in this case as well?

```
snowflake-delete:
    name: Delete Snowflake development environment
    if: ${{ github.event.ref != 'main' && github.event.ref != 'staging' }}
```

Because usually we use `if: github.ref_name != 'main'`

Reply:
> it just spits out the exact same thing as github.ref_name

This appears to be the case. There is no such thing as `github.event.ref_name`. See the available properties here: https://docs.github.com/en/rest/using-the-rest-api/github-event-types?apiVersion=2022-11-28#event-object-common-properties

Reply:
Right, `github.ref_name` exists here: https://docs.github.com/en/actions/learn-github-actions/contexts#github-context